screens/Media.js

```diff
@@ -134,22 +136,40 @@ const MediaScreen = () => {
       </Text>
       {isTheWeekend && (
         <>
-          <Text style={styles.sectionHeaderText}>WATCH NOW</Text>
-          <WebView
+          {/* <Text style={styles.sectionHeaderText}>WATCH NOW</Text> */}
+          <TouchableHighlight
+            onPress={() => {
+              Amplitude.logEventWithProperties('mobileEngagementAction', {
+                app: 'mobile',
+                connect: 'watch live',
+              });
+              WebBrowser.openBrowserAsync(
+                'https://echochurchlive.churchonline.org',
+                { toolbarColor: Colors.darkestGray }
+              );
+            }}
+          >
+            <LiveCard
+              style={styles.largeCard}
+              width={styles.largeCard.width}
+              height={styles.largeCard.height}
+            />
+          </TouchableHighlight>
+          {/* <WebView
```

Comment:
you can just remove this 😅 we can always find it in git history if we need to

Reply:
Lol yeah. Oh. I forgot about that...